units/en/conclusion/conclusion.mdx

```diff
@@ -1,5 +1,14 @@
 # Conclusion
 
-This course is in-progress and has not yet concluded!
+Thank you for participating in the Machine Learning for 3D Course!
 
-To receive updates as the course releases, sign up for the course mailing list [here](https://mailchi.mp/911880bcff7d/ml-for-3d-course).
+This has been a high-level overview of what's going on at the intersection of Machine Learning and 3D. For further exploration:
+
+- Keep up with [3D Arena](https://huggingface.co/spaces/dylanebert/3d-arena) for the latest projects.
+- Explore open-source projects like [InstantMesh](https://github.com/TencentARC/InstantMesh).
+
+![3D Arena](https://huggingface.co/datasets/dylanebert/ml-for-3d-course/resolve/main/3d-arena.png)
+
+As the field evolves rapidly, it's easy to get overwhelmed. Stay tuned for more accessible tools and resources as these projects mature.
```

Comment:
Is there a Discord channel/ track where people can discuss the course, we can link that here, we did this with audio course and people still use it to discuss things.